packages/coachmark/README.md

```diff
@@ -66,17 +86,11 @@ Coach marks can include an `<sp-action-menu>`, which appears at the top right of
 </sp-coachmark>
 ```
 
-## User Action Dependent
-
-User action-dependent coachmarks are designed to guide users based on their interactions within your application. In such cases, there is no "Next Step" button, as the coachmark progresses when the user takes a specific action. This allows users to learn by doing, rather than simply reading instructions. The coachmark remains until the user performs the required action or takes an alternative route in the tour, such as skipping, restarting, or moving back to a previous step.
-
-Inside the `<sp-coachmark>`, add the content and instructions for the coachmark in the `<sp-coachmark>`. You can also define primary and secondary CTA buttons for user interaction.
+#### User Action Dependent Coachmarks
```

Comment:
Can we place this under a "Behaviors" section to make it consistent with other docs?

See https://ttomar-coachmark-docs-audit--spectrum-wc.netlify.app/components/alert-banner/#behaviors